scripts/build.py

```diff
@@ -1265,7 +1283,10 @@ def main(args=None):
         minify_third_party_libs(THIRD_PARTY_GENERATED_DEV_DIR)
         if not options.minify_third_party_libs_only:
             hashes = generate_hashes()
-            build_using_webpack()
+            if not options.deparallelize_terser:
```

Comment:
This is an unnecessary if-condition negation, change it so that its just `if options.deparallelize_terser` and adjust the statements accordingly.

Reply:
Done